modules/math/test/geospatial/ellipsoid.spec.js

```diff
@@ -95,7 +95,6 @@ test('Ellipsoid#geodeticSurfaceNormal works with a result parameter', t => {
   const ellipsoid = Ellipsoid.WGS84;
   const result = new Vector3();
   const returnedResult = ellipsoid.geodeticSurfaceNormal(spaceCartesian, result);
-  t.ok(returnedResult === result);
```

Comment:
This test is there to verify that you are actually returning the result object if supplied.